src/ConnectorEdgeModule/moduleSettings.example.json

```diff
@@ -0,0 +1,82 @@
+﻿{
+  "RootDirectory": null,
```

Comment:
Do we still need these 2 settings. 
RootDirectory
KeyVaultBaseUrl (this is platform specific (Azure) - not sure how this fits into OMP)

I believe these were for the SettingsReader nuget package?

Reply:
Not sure either. I will remove it